### PR TITLE
Feat checkbox

### PR DESCRIPTION
**Pull request by LDeLaRosa13**
closes issue #14 

What did I do? 
-Added dietary options to query/mutation
-deleted console logs
-Add styling to check boxes (needs finished)
-Updated useStates and onChange action

Testing Notes/Extras:

Did this break anything? 

- [x ]  No
- [ ]  Yes

Type of change

- [ x]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [ x]  The code runs locally
- [x ]  The code is DRY. If it's not, I tried my best

### CODE REVIEW

File: src/Components/Form/Form.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import "./Form.css";
 import { useState } from "react";
 import { gql, useMutation } from "@apollo/client";
-import foodieLogo from '../.././images/foodie-brain-logo.png'
```
Comment (paulina-isabel) [inline src/Components/Form/Form.js:4]: Good catch here.

File: src/Components/Form/Form.js [hunk 2]
```diff
@@ -10,22 +10,43 @@ const POST_REVIEW = gql`
     $description: String!
     $lat: String!
     $lng: String!
+    $dairyFree: Int = 0
```
Comment (paulina-isabel) [inline src/Components/Form/Form.js:13]: Is there a reason dairyFree has a `= 0` after `Int` and the others don't?

**Review by paulina-isabel — APPROVED** [review]
Good use of those ternaries! Looking great.

File: src/Components/Form/Form.js (hunk 2 above)
Comment (LDeLaRosa13) [reply src/Components/Form/Form.js:13]: Yeah, that was a leftover piece from the trial and error with Weston! It's removed now. Good catch!

**Pull request merged**